install/kubernetes/helm/istio/charts/mixer/templates/deployment.yaml

```diff
@@ -427,7 +427,6 @@ spec:
         chart: {{ template "mixer.chart" $ }}
         heritage: {{ $.Release.Service }}
         release: {{ $.Release.Name }}
-        security.istio.io/mtlsReady: "true"
```

Comment:
please make sure this works when `--set global.controlPlaneSecurityEnabled=true` and automtls is enabled.

Reply:
done.

Reply:
mixer always has a DR to configure the client side config.